src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -9,9 +9,11 @@
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
+    public static final Prefix PREFIX_EXPENSE = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_TITLE = new Prefix("t/");
     public static final Prefix PREFIX_PARTICIPANT = new Prefix("p/");
```

Comment:
Actually I wonder why do we use participant instead of name? Is there a difference

Reply:
Well I used participant because thats what we call people that takes part in an activity.. Honestly it doesn't matter.

Reply:
it's also in our user guide to use p/ to indicate searching for participants

Reply:
@podocarp @daekoon @liakify It seems that multiple arguments share the same prefix string. Even though this causes no error, do we want to give each argument type a distinct prefixes (may require some 2-letter prefix strings)?